js/src/pages/attribute-mapping/index.test.js

```diff
@@ -136,6 +136,10 @@ jest.mock( '~/components/tours/rebranding-tour', () =>
 	jest.fn().mockReturnValue( null ).mockName( 'RebrandingTour' )
 );
 
+jest.mock( '~/components/experience-rating-banner', () =>
```

Comment:
@ankitrox Why we need to mock that component please?

Reply:
We need to mock this component because it throws the error for this line

```
const { set } = useDispatch( preferencesStore );
```

As it could not get the `set` method and it is undefined.

We have separate tests for this banner, so I think it is fine if we mock it here.